lib/Template.js

```diff
@@ -0,0 +1,54 @@
+try {
+  module.exports = require('./es6/Template')
+} catch (ignored) {
+  // ES6 code failed to load.
+  //
+  // This happens in Node runtimes with versions < 6.
+  // Since those runtimes won't parse template tags, we
+  // fallback to an equivalent API that assumes no calls
+  // are template tag calls.
+  //
+  // Clients that need to work on older Node runtimes
+  // should not use any part of this API except
+  // calledAsTemplateTagQuick unless that function has
+  // returned true.
+
+  module.exports = function (staticStrings) {
+    // This might be reached if client code is transpiled down to
+    // ES5 but this module is not.
+    throw new Error('ES6 features not supported');
+  };
+  /**
+   * @param {*} firstArg
+   * @param {number} nArgs
+   *
+   * @return {boolean} always false in ES<6 compatibility mode.
+   */
+  module.exports.calledAsTemplateTagQuick = function (firstArg, nArgs) {
+    return false;
+  };
+
+  function stringWrapper() {
```

Comment:
What does this do, exactly? How is it different from the `toSqlString` already implemented here?

Reply:
So the difference is that you want to use instanceof I see. Well, that won't work, so there doesn't sound like a difference then?

Reply:
Removed in favor of SqlString.raw and SqlString.identifier.